{"name": "test_register[Create user - username Numeric]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:41 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:41 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "9012eca1-c99b-40a8-8c9c-3889b3deefef-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Create user - username Numeric', 'Test Type': 'Negative', 'Request_Body': {'firstName': 'QZetUjdBWTxbDJDgyuvEXAVvvU', 'lastName': 'YxfGKfVUjlTdSiKJHiPJKidYBU', 'username': '40836829', 'password': 'oYch3515'}}"}], "start": 1670168725248, "stop": 1670168726042, "uuid": "2460b9f0-e0cf-4460-8b4a-9df46aa7be00", "historyId": "3c0157d3ab6b49e8e2fdf436b438a9d6", "testCaseId": "bd51a8de3249e024de618e4e17a0a343", "fullName": "tests.test_api.TestApi#test_register", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}